hwpf/fapi2/src/plat/plat_error.C

```diff
@@ -0,0 +1,74 @@
+/* IBM_PROLOG_BEGIN_TAG                                                   */
```

Comment:
either add the correct prologue or remove from the list.

Reply:
removed prologue